src/index.spec.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { mount, render } from 'enzyme'
 import { spy, stub } from 'sinon'
+import getDataTransferFiles from './getDataTransferItems'
```

Comment:
Dropzone.onDragEnter now work with async getDataTransferItems, how can i mock it with sync variant in test?)